epub33/fxl-a11y/index.html

```diff
@@ -67,36 +67,19 @@ <h3>Overview</h3>
 
```

Comment:
I think we should add the statement of what is an Accessible FXL

Reply:
And also a section on different techniques used for making FXL (HTML, SVG, raster images)